### PR TITLE
Alphabetized package lists

### PR DESCRIPTION
**Pull request by jgbradley1**
Alphabetize all package.json lists

Signed-off-by: Joshua Bradley <jgbradley1@gmail.com>

### CODE REVIEW

File: theia-java-docker/next.package.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+        "@theia/outline-view": "next",
+        "@theia/preferences": "next",
         "@theia/search-in-workspace": "next",
-        "@theia/json": "next",
+        "@theia/terminal": "next",
         "@theia/textmate-grammars": "next",
-        "@theia/editor-preview": "next"
+        "@theia/tslint": "next",
+        "@theia/typescript": "next",
+        "typescript": "latest",
```
Comment (marcdumais-work) [inline theia-java-docker/next.package.json]: remove trailing comma

File: theia-swift-docker/latest.package.json [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-        "@theia/callhierarchy": "latest",
-        "@theia/merge-conflicts": "latest",
         "@theia/search-in-workspace": "latest",
-        "@theia/json": "latest",
+        "@theia/terminal": "latest",
         "@theia/textmate-grammars": "latest",
-        "@theia/mini-browser": "latest",
-        "@theia/plugin-ext-vscode": "latest"
+        "@theia/typescript": "latest",
+        "typescript": "latest",
```
Comment (marcdumais-work) [inline theia-swift-docker/latest.package.json]: remove trailing comma

File: theia-swift-docker/next.package.json [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
         "@theia/search-in-workspace": "next",
-        "@theia/json": "next",
+        "@theia/tslint": "next",
+        "@theia/terminal": "next",
         "@theia/textmate-grammars": "next",
-        "@theia/mini-browser": "next",
-        "@theia/editor-preview": "next",
-        "@theia/plugin-ext-vscode": "next"
+        "@theia/typescript": "next",
+        "typescript": "latest",
```
Comment (marcdumais-work) [inline theia-swift-docker/next.package.json]: remove trailing comma

File: yangster-docker/next.package.json [hunk 4]
```diff
@@ -11,18 +10,19 @@
...
         "@theia/process": "next",
+        "@theia/search-in-workspace": "next",
         "@theia/terminal": "next",
         "@theia/textmate-grammars": "next",
         "@theia/tslint": "next",
         "@theia/typescript": "next",
         "@theia/workspace": "next",
         "theia-yang-extension": "next",
-        "@theia/search-in-workspace": "next"
+        "typescript": "latest",
```
Comment (marcdumais-work) [inline yangster-docker/next.package.json]: remove trailing comma

**Comment by jgbradley1** [comment]
Done

**Pull request merged**